priv/repo/migrations/20180101181301_create_authorizations.exs

```diff
@@ -17,11 +17,10 @@ defmodule Cadet.Repo.Migrations.CreateAuthorizations do
 
     create(unique_index(:authorizations, [:provider, :uid]))
     create(index(:authorizations, [:provider, :token]))
+    create(index(:authorizations, [:user_id]))
   end
 
   def down do
-    drop(index(:authorizations, [:provider, :uid]))
-    drop(index(:authorizations, [:provider, :token]))
```

Comment:
I think this line should remain and you should add `drop` for the other index and unique_index

Reply:
indexes drop with the table by default
see: https://www.postgresql.org/docs/8.2/static/sql-droptable.html

Reply:
Cool